src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsTrackingPixels.java

```diff
@@ -89,6 +96,23 @@ public void adsTrackingPixelSentCuratedMainPages(String wiki, String page, Strin
     assertTrackingPixelsSent(adsBaseObject, pixelUrls);
   }
 
+  @NetworkTrafficDump
+  @Test(
+          groups = "AdsTrackingPixelsAuthPage",
+          dataProviderClass = AdsDataProvider.class,
+          dataProvider = "adsTrackingPixelsSentAuthPage"
+  )
+  @InBrowser(
+          browser = Browser.CHROME,
+          emulator = Emulator.GOOGLE_NEXUS_5
+  )
+  public void adsTrackingPixelSentAuthPage(String wiki, String page, String[] pixelUrls) {
```

Comment:
@kvas-damian It's not a copy because of the @InBrowser annotation which identifies specific test environment for the AuthPage test. The test body is a copy of the adsTracingPixelSentCuratedMainPages test but extracting it to separate method would not spare any lines of code: 8 vs 1 + 1 + 6. I would do it however I can't think of any good name for such method, as the generic adsTrackingPixelSent name is already in use in this class.